prover/prover_cli/src/commands/stats.rs

```diff
@@ -0,0 +1,63 @@
+use anyhow::Context;
+use chrono::{self, NaiveTime};
+use clap::{Args, ValueEnum};
+use zksync_basic_types::prover_dal::ProofGenerationTime;
+use zksync_db_connection::connection_pool::ConnectionPool;
+use zksync_prover_dal::{Prover, ProverDal};
+
+use crate::cli::ProverCLIConfig;
+
+#[derive(ValueEnum, Clone)]
+enum StatsPeriod {
+    Day,
+    Week,
+}
+
+#[derive(Args)]
+pub(crate) struct Options {
+    #[clap(
+        short = 'p',
+        long = "period",
+        help = "Specify the time frame to look for stats",
+        default_value = "day"
+    )]
+    period: StatsPeriod,
+}
+
+pub(crate) async fn run(opts: Options, config: ProverCLIConfig) -> anyhow::Result<()> {
+    let prover_connection_pool = ConnectionPool::<Prover>::singleton(config.db_url)
+        .build()
+        .await
+        .context("failed to build a prover_connection_pool")?;
+    let mut conn = prover_connection_pool
+        .connection()
+        .await
+        .context("failed to get connection from pool")?;
+
+    let start_date = match opts.period {
+        StatsPeriod::Day => chrono::offset::Local::now().date_naive(),
```

Comment:
Am I getting this right, that we can receive only stats from the beginning of today till now(or for the whole week)? It makes not much sense to me to be honest. As a first iteration of adding this command it should be okay, but I would love to see stats for a specific date

Reply:
I am not that familiar with clap, but probably it is possible to add one more enum option like CustomDate

Reply:
Good, let's put this somewhere in the backlog then